README.md

```diff
@@ -1,4 +1,151 @@
+
+## End-to-End Test Suite for the CS3 APIs
+
+**This tool will receive a lot of changes before version 1.0.0**
+
+The cs3api-validator is a tool to test implementations of the [CS3Apis](https://github.com/cs3org/cs3apis). It works as a standalone software which only needs the address of a running cs3api provider.
+
+## Purpose
+
+### BDD (Behavior driven development)
+
+The cs3api-validator can be run locally in a development phase to develop against a well defined set of basic API operations. It has no external dependencies and runs human readable gherkin test scenarios. This helps to understand the behavior of the CS3APIs being an additional way of documenting the API in combination with the specification.
+
+### Litmus Testing
+
+This tool makes it possible to confirm that an implementation of the CS3APIs is compliant to the spec and fulfills the basic operations. This helps the CS3 community to keep different implementations in sync and foster compatibility between them.
+
+## Contributions
+
+This is a community driven Open Source Project. We welcome contributions from everyone and we're ready to support you if you have the enthusiasm to contribute.
+
+Please use the [issue tracker](https://github.com/owncloud/cs3api-validator/issues) to report problems or propose changes.
+
+## Getting help
+
+We have a [Public Chat](https://talk.owncloud.com/) where you can chat with other community members and developers.
+
+Here are some useful channels to try:
+
+- [#ocis](https://talk.owncloud.com/channel/ocis) - Next Generation ownCloud Architecture
+- [#general](https://talk.owncloud.com/channel/general) - General Topics around File Sync & Share
+
+## Developing
+
+### Quick start
+```shell
+git clone git@github.com:owncloud/cs3api-validator.git
```

Comment:
```
$ git clone git@github.com:owncloud/cs3api-validator.git
Cloning into 'cs3api-validator'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
$ git clone https://github.com/owncloud/cs3api-validator.git
Cloning into 'cs3api-validator'...
remote: Enumerating objects: 163, done.
remote: Counting objects: 100% (163/163), done.
remote: Compressing objects: 100% (113/113), done.
remote: Total 163 (delta 79), reused 116 (delta 44), pack-reused 0
Receiving objects: 100% (163/163), 97.49 KiB | 486.00 KiB/s, done.
Resolving deltas: 100% (79/79), done.
```

What am I doing wrong. The suggested command gives me errors, but I can clone https://github.com/owncloud/cs3api-validator.git

Reply:
seems that http works and ssh not.